functions/src/project/delete-project.ts

```diff
@@ -29,6 +30,8 @@ async function deleteProjectRepositories(projectUid: string, project: ProjectMod
               }
 
               t.update(repoRef, { projects: repoData.projects });
+            
+              deleteRepoBuilds(repositoryUid);
```

Comment:
this is a promise and should be handled. I fixed this, may be it was in a commit in the old PR

Reply:
This has been commented in the https://github.com/DashboardHub/PipelineDashboard/pull/1604/files